src/gui/guiPasswordChange.cpp

```diff
@@ -215,12 +215,14 @@ bool GUIPasswordChange::processInput()
 bool GUIPasswordChange::OnEvent(const SEvent &event)
 {
 	if (event.EventType == EET_KEY_INPUT_EVENT) {
+		// clang-format off
```

Comment:
they should not be indented

Reply:
As far as I can see this is a line continuation and should be double indented. The linter, however, suggests I add some spaces to indent it further. Please feel free to fix that to your liking on merge, I am done with it now :P

Reply:
I agree that indented is better to visually preserve the indentation of the code block.
The linter unfortunately asks for double-tab indentation often, which is wrong.
So it's ok as-is.